app/src/ingest_policy_pdfs.py

```diff
@@ -22,11 +32,93 @@ def _ingest_policy_pdfs(
     embedding_model = app_config.sentence_transformer
     for file in file_list:
         if file.endswith(".pdf"):
+            with smart_open_file(file, "rb") as fin:
+                extract_text_to_fp(fin, output_string, laparams=LAParams(), output_type="text")
```

Comment:
Could you use [extract_text](https://pdfminersix.readthedocs.io/en/latest/reference/highlevel.html#extract-text) here so that this becomes `output_string = extract_text(...` (rather than having to declare a global `output_string = StringIO()`